src/devices/gledopto.ts

```diff
@@ -969,6 +969,14 @@ const definitions: DefinitionWithExtend[] = [
         description: 'Filament LED light bulb E27 G95 7W pro',
         extend: [gledoptoLight({colorTemp: {range: [158, 495]}})],
     },
+    {
+        zigbeeModel: ['GL-SD-301P'],
+        model: 'GL-SD-301P',
+        vendor: 'Gledopto',
+        description: 'Zigbee triac AC dimmer',
+        extend: [gledoptoLight({})],
+        meta: {disableDefaultResponse: true},
```

Comment:
Is this line really needed?

Reply:
yes, I am an employee of this company, and my boss told me to write this. 
Customers require the Z2M to support 'GL-SD-301P'

Reply:
I meant the line `meta: {disableDefaultResponse: true},`, if you remove it, does the device it still work?

Reply:
> I meant the line `meta: {disableDefaultResponse: true},`, if you remove it, does the device it still work?

yes.